tests/schema/core/issues/issue-514.json

```diff
@@ -34,13 +34,16 @@
             },
             "b": {
               "type": "number"
+            },
+            "c": {
```

Comment:
Adding this third property to test that optional properties can occur independently of one another.